src/components/SearchBar/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React, {Fragment} from 'react';
+import {StyledInput} from './styles';
+
+interface Props {
+    searchTerm: string;
+    setSearchTerm: (value: string) => void,
+    isLoading: boolean,
+    foundUsers: boolean
+}
+
+const SearchBar = ({searchTerm = '', setSearchTerm, isLoading, foundUsers = false}: Props) => {
+    const handleSearch = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setSearchTerm(event.target.value);
+    };
+
+    return isLoading ? null : (
+        <React.Fragment>
+            <StyledInput type="text" placeholder="Search" value={searchTerm} onChange={handleSearch} />
+            {!foundUsers && !!searchTerm.length && <div>Nothing found 😔.</div>}
```

Comment:
Having the message on this component is a way to not repeat the message or check repeated on multiple components, and only having the boolean as a prop is a way to allow each page to have its own data it is searching for ( Ex. In teams page it is showing all teams, and on TeamOverview there are users and team leaders)